simplified-ui-catalog/src/main/java/org/librarysimplified/ui/catalog/CatalogBookDetailFragment.kt

```diff
@@ -912,10 +895,6 @@ class CatalogBookDetailFragment : Fragment(R.layout.book_detail) {
           }
         )
       )
-    } else if (!createPreviewButton) {
-      // add spaces on both sides if there aren't any other buttons
-      this.buttons.addView(this.buttonCreator.createButtonSizedSpace(), 0)
-      this.buttons.addView(this.buttonCreator.createButtonSizedSpace())
```

Comment:
I think that removing these two buttons is causing [this](https://github.com/ThePalaceProject/android-core/assets/79104027/83f9d8d5-d241-41d5-af69-eaabe0f94836). If you just remove the condition of the `else if` and keep the rest, it should be fixed. Also, I think you can remove the `bookPreviewStatus` as argument of this method since it's no longer used

Reply:
Oops!